lib/pf/radius.pm

```diff
@@ -790,6 +791,9 @@ sub switch_access {
     my $connection_type = $connection->attributesToBackwardCompatible;
     my $connection_sub_type = $connection->subType;
     my $password = $radius_request->{'User-Password'};
+    if (exists($radius_request->{'PacketFence-UserPassword'})) {
```

Comment:
Should we not rename `password` variable to distinguish:
- unencoded variable coming from `User-Password`
- encoded variable coming from `PacketFence-UserPassword` 

It's just a suggestion because the line 793 is confusing for me.